crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/binary.py

```diff
@@ -424,3 +424,8 @@
     expression } bbbbbbbbbbbbbbbbbbbbbbbb" + (
     yyyyyyyyyyyyyy + zzzzzzzzzzz
 )
+
+# This is not a multiline f-string, but the expression is too long so it should be
+# wrapped in parentheses.
+f"hellooooooooooooooooooooooo \
```

Comment:
Could we add some tests in assignment positions and with implicit concatenated strings (especially with inline comments) just to make sure strings with line continuations are handled correctly in those positions too